### ISSUE TITLE
How to custom native add

### ISSUE THREAD

**Issue by zaouifahd** [floatinghotpot/cordova-plugin-facebookads#31]
Great job.

I need costum native add how can I do it?, can i add class 

thank you lot

**Comment by floatinghotpot** [comment]
Please read this example:
https://github.com/floatinghotpot/cordova-plugin-facebookads/tree/master/test

Just get the Ad resources (picture/text) from onAdLoaded event, then display it in your HTML.

**Comment by zaouifahd** [comment]
Yes, i have already read and i have try it, but i dont know how change style or add class, and the value of rating always return zero. 
And i have other question, can i use horizontal scroll with multiple ads ? ( Horizontal Scroll View )
link: https://developers.facebook.com/docs/audience-network/ios/nativescroll

Thank you

**Comment by floatinghotpot** [comment]
In a normal native app, you can display ad in a native view.

In a cordova app, you can display ad in a HTML DOM, like DIV. 

You can create multiple native ad, if pass different ad unit id.
